src/utils/convert3Dto4D.m

```diff
@@ -0,0 +1,192 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function convert3Dto4D
+%
+% Short description of what the function does goes here.
+%
+% USAGE::
+%
+%   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
+%
+% :param argin1: (dimension) obligatory argument. Lorem ipsum dolor sit amet,
+%                consectetur adipiscing elit. Ut congue nec est ac lacinia.
+% :type argin1: type
+% :param argin2: optional argument and its default value. And some of the
+%               options can be shown in litteral like ``this`` or ``that``.
+% :type argin2: string
+% :param argin3: (dimension) optional argument
+%
+% :returns: - :argout1: (type) (dimension)
+%           - :argout2: (type) (dimension)
+%
+% .. todo:
+%
+%    - expand to run through multiple subjs ans groups
+%      (https://stackoverflow.com/questions/8748976/list-the-subfolders-in-a-folder-matlab-only-subfolders-not-files)
+%    - generalize how to retrieve RT from sidecar json file
+%    - saveMatlabBatch(matlabbatch, ['3Dto4D_dataType-' num2str(dataType) '_RT-' num2str(RT)], opt, subID);
+
+
+% Set the folder where sequences folders exist
+optSource.sourceDir = '/Users/barilari/Desktop/DICOM_UCL_leuven/renamed/sub-pilot001/ses-002/MRI';
+
+% List the sequences that you want to skip (folder name)
+optSource.sequenceToIgnore = {'001-AAHead_Scout_32ch-head-coil', ...
+    '002-AAHead_Scout_32ch-head-coil_MPR_sag' ,...
+    '003-AAHead_Scout_32ch-head-coil_MPR_cor', ...
+    '004-AAHead_Scout_32ch-head-coil_MPR_tra', ...
+    '005-b1map_sag_p2', ...
+    '006-b1map_sag_p2', ...
+    '007-t1_mprage_sag_p2_0.70mm', ...
+    '011-gre_field_mapping_2mm_eufind', ...
```

Comment:
Might be possible to make this easier to deal with but replacing the `strcmp` line 81 with a `ismember`.

This would allow to detect a certain pattern like `gre_field_mapping` and not have to write all the instances.

Reply:
done